jmclient/jmclient/configure.py

```diff
@@ -541,8 +540,9 @@ def validate_address(addr):
 def is_burn_destination(destination):
     return destination == _BURN_DESTINATION
 
+#Donation code currently disabled.
+"""
```

Comment:
Might be better to just delete all this donation code, its been unused for years

Reply:
Agree with this.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/commit/adbbb24dae155334aa9e50a72f5b4397db175450